LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
OMG, This is the 10th time you created a license and removed my readme!! Stop doing that, revert all those changes.